src/OmniSharp.Roslyn/OmniSharpWorkspace.cs

```diff
@@ -340,7 +340,8 @@ public IEnumerable<Document> GetDocuments(string filePath)
         {
             return CurrentSolution
                 .GetDocumentIdsWithFilePath(filePath)
-                .Select(id => CurrentSolution.GetDocument(id));
+                .Select(id => CurrentSolution.GetDocument(id))
+                .OfType<Document>();
```

Comment:
Since the .cshtml files will have valid DocumentIds but not be part of any Project's Documents null can be returned, the `OfType` will filter out the null values.